wallet-sdk/src/test/kotlin/org/stellar/walletsdk/CustomerManagementTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.stellar.walletsdk
+
+import io.ktor.client.plugins.*
+import io.ktor.http.*
+import kotlin.test.assertNotNull
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.stellar.walletsdk.horizon.SigningKeyPair
+
+class CustomerManagementTest {
+  private val wallet =
+    Wallet(
+      StellarConfiguration.Testnet,
+      ApplicationConfiguration { defaultRequest { url { protocol = URLProtocol.HTTP } } }
+    )
+  private val anchor = wallet.anchor("https://$AUTH_HOME_DOMAIN")
+  val keypair =
+    SigningKeyPair.fromSecret("SDGEPZ5QOQ24XGCVA274ZL43OKI6NND5CSR4XD4UH6QT3AA33P66FAJW")
+
+  @BeforeEach
+  fun setup() {
+    System.setProperty("IS_TEST_RUNNING", "true")
```

Comment:
Why do you need this variable?

Reply:
In Anchor Platform toml example it use `sep12` endpoint instead `kyc`. So I set `IS_TEST_RUNNING` to identify when I need to set the `kyc` endpoint. You can check [here](https://github.com/stellar/kotlin-wallet-sdk/blob/490df5aa585b9360de3612121f54fe8e6ecc8348/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/customer/Customer.kt#L75)

Reply:
Solved this issue by getting the KYC server from the toml file.